### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by ride90**

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    environment:
+      - MEDIA_STORAGE=filesystem
+      - FS_MEDIA_STORAGE_PATH=/opt/media
+      - RABBIT_MQ_URL=pyamqp://guest@rabbit//
+      - MONGO_HOST=mongodb
+    volumes:
+      - server_data:/data/db:rw
+volumes:
+  mongodb_data: {}
+  server_data: {}
```
Comment (ride90) [inline docker/docker-compose.yml]: For now video/thumbnail files are stored in named volume. If needed it's possible to explore these files by entering a container using `exec -it`.
As an option, I can mount a host path for it.

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## system-wide dependencies
+RUN apt-get update && \
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+build-essential git ffmpeg bc \
+python3 python3-dev python3-pip python3-setuptools python3-wheel
+
+# install video-server app
+WORKDIR /opt/video-server-app
+RUN git clone https://github.com/superdesk/video-server-app.git .
```
Comment (petrjasek) [inline docker/Dockerfile]: I wonder why not use `ADD` if this file is already in the repo

Comment (ride90) [reply docker/Dockerfile]: I think in this case I should move `Dockerfile` outside of the `docker` folder.
`ADD` or `COPY` would say `Forbidden path outside the build context`

Comment (petrjasek) [reply docker/Dockerfile]: yep right

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      - RABBITMQ_DEFAULT_USER=guest
+      - RABBITMQ_DEFAULT_PASS=guest
+  mongodb:
+    image: mongo:3
+    container_name: video-server-mongodb
+    volumes:
+      - mongodb_data:/data/db
+  server:
+    build: .
+    command: honcho start
```
Comment (petrjasek) [inline docker-compose.yml]: you have have that in Dockerfile so it's usable on its own:
```
ENTRYPOINT ["honcho"]
CMD ["start"]
```

File: README.md [hunk 4]
```diff
@@ -21,4 +23,31 @@ honcho start
...
- Using Honcho is not required, you can run processes listed in `Procfile` with another tool (e.g. Supervisor).
\ No newline at end of file
+ Using Honcho is not required, you can run processes listed in `Procfile` with another tool (e.g. Supervisor).
+ 
+### Run with docker
+ 
+#### docker-compose
+```
+git clone https://github.com/superdesk/video-server-app.git
+docker-compose up -d
```
Comment (petrjasek) [inline README.md:33]: I think you have to do `cd video-server-app` in between

Comment (ride90) [reply README.md:33]: right

**Review by petrjasek — APPROVED** [review]

**Pull request merged**